app.rb

```diff
@@ -9,51 +9,41 @@
 KAFKA_TOPIC = ENV.fetch('KAFKA_TOPIC', 'messages')
 GROUP_ID = ENV.fetch('KAFKA_CONSUMER_GROUP', 'heroku-kafka-demo')
 
-def with_prefix(name)
-  "#{ENV['KAFKA_PREFIX']}#{name}"
-end
```

Comment:
Why was this removed? `KAFKA_PREFIX` is still required for the topic and consumer group.

Reply:
That was a mistake on my part and it has been re-added.